src/config/vega.ts

```diff
@@ -22,6 +23,20 @@ export type NetworkConfig = {
   [N in Networks]: string[];
 };
 
+const splitFilter = (a: string) => a.split(',').filter(a => a.length > 0);
+const getValidatorNetNodesFromOptionalEnvironmentVariables = () => {
+  const validatorUrls = process.env.VALIDATORNET_URLS || ""
```

Comment:
It looks like this breaks if both `VALIDATORNET_URLS` and `VALIDATORNET_URLS_WITH_GRAPHQL` are undefined. Is my understanding correct? If so, can you update the `README` so that it's clear that one (or both?) of these need to be defined.